addons/knowledge/static/src/js/knowledge_renderers.js

```diff
@@ -112,20 +115,29 @@ const KnowledgeFormRenderer = FormRenderer.extend({
         if ($li.closest('ul.o_tree_private').length !== 0) {
             params.private = true;
         }
-
-        const result = await this._rpc({
-            model: 'knowledge.article',
-            method: 'move_to',
-            args: [
-                $li.data(key),
-            ],
-            kwargs: params,
+        // TODO FABA: Waiting for jbn to complete this function
+        const moveArticle = async () => {
+            const result = await this._rpc({
+                model: 'knowledge.article',
+                method: 'move_to',
+                args: [
+                    $li.data(key),
+                ],
+                kwargs: params,
```

Comment:
don't forget to rebase, $li and key are not defined (my fault) but that has been corrected.